Source/Engine/Core/Types/Variant.cpp

```diff
@@ -1578,8 +1590,10 @@ Variant::operator void*() const
     case VariantType::Blob:
         return AsBlob.Data;
     case VariantType::ManagedObject:
-#if USE_MONO
-        return AsUint ? mono_gchandle_get_target(AsUint) : nullptr;
+#if USE_NETCORE
+        return AsUint64 ? MUtils::GetGCHandleTarget(AsUint64) : nullptr;
```

Comment:
Maybe we could just use `AsUint64` for GCHandleTarget storage for both Mono and NetCore versions? (simplify code)